script/install.sh

```diff
@@ -946,7 +946,7 @@ verify() {
     path="$1"
     urls=("${@:2}")
     : "${GPG:=$(find_gpg)}"
-    if [ "$GPG" = x ]; then
+    if [ -z "$GPG" ]; then
```

Comment:
I have no idea what `[ "$GPG" = x ]` was supposed to check for... perhaps the intention was to perform this check instead: `[ "$GPG" != gpg ] && [ "$GPG" != gpg2 ]`?

Reply:
I think it was supposed to be `[ x$GPG = x ]`, which I think is an antiquated way to check for an empty variable.